client/extensions/woocommerce/app/reviews/review-card.js

```diff
@@ -155,7 +155,6 @@ class ReviewCard extends Component {
 							<div
 								className="reviews__content"
 								dangerouslySetInnerHTML={ { __html: review.review } } //eslint-disable-line react/no-danger
-								// Also used in `comment-detail/comment-detail-comment.jsx` to set the rendered content correctly
```

Comment:
I didn't know about this!
Instead of removing it, though, I'd change it to point to `comment-content.jsx`:
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/comments/comment/comment-content.jsx#L105

Reply:
Updated in 0a613e41058b64b8827af927f5ee7c2a68dabcbd.